### PR TITLE
[zh-cn]: update the translation of svg `<font>` and `<font-face-*>` element

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/web/svg/element/font-face-format/index.md [hunk 1]
```diff
@@ -1,33 +1,35 @@
 ---
-title: font-face-format
+title: <font-face-format>
 slug: Web/SVG/Element/font-face-format
+l10n:
+  sourceCommit: 3a1ef2abc8233835f0b0cc73afaf36e44edaf4a1
```
Comment (JasonLamv-t) [inline files/zh-cn/web/svg/element/font-face-format/index.md:5]: Seems like the commit hash incorrecet here. Pls double check.
```suggestion
  sourceCommit: b4f998244660723175f8e06b5d77f68cfb1d1f1a
```

File: files/zh-cn/web/svg/element/font-face-name/index.md [hunk 2]
```diff
@@ -1,33 +1,35 @@
 ---
-title: font-face-name
+title: <font-face-name>
 slug: Web/SVG/Element/font-face-name
+l10n:
+  sourceCommit: 3a1ef2abc8233835f0b0cc73afaf36e44edaf4a1
```
Comment (JasonLamv-t) [inline files/zh-cn/web/svg/element/font-face-name/index.md:5]: Same here.

File: files/zh-cn/web/svg/element/font-face-src/index.md [hunk 3]
```diff
@@ -1,33 +1,33 @@
 ---
-title: font-face-src
+title: <font-face-src>
 slug: Web/SVG/Element/font-face-src
+l10n:
+  sourceCommit: 3a1ef2abc8233835f0b0cc73afaf36e44edaf4a1
```
Comment (JasonLamv-t) [inline files/zh-cn/web/svg/element/font-face-src/index.md:5]: Ditto.

File: files/zh-cn/web/svg/element/font-face-uri/index.md [hunk 4]
```diff
@@ -1,34 +1,35 @@
 ---
-title: font-face-uri
+title: <font-face-uri>
 slug: Web/SVG/Element/font-face-uri
+l10n:
+  sourceCommit: 3a1ef2abc8233835f0b0cc73afaf36e44edaf4a1
```
Comment (JasonLamv-t) [inline files/zh-cn/web/svg/element/font-face-uri/index.md:5]: Ditto.

File: files/zh-cn/web/svg/element/font/index.md [hunk 5]
```diff
@@ -1,44 +1,42 @@
 ---
-title: font
+title: <font>
 slug: Web/SVG/Element/font
+l10n:
+  sourceCommit: 3a1ef2abc8233835f0b0cc73afaf36e44edaf4a1
```
Comment (JasonLamv-t) [inline files/zh-cn/web/svg/element/font/index.md:5]: Ditto.

**Review by JasonLamv-t — APPROVED** [review]
Beside the hash, else things looks fine to me. Thanks! @T34-active :heart:

**Comment by T34-active** [comment]
https://github.com/mdn/content/commit/3a1ef2abc8233835f0b0cc73afaf36e44edaf4a1

File: files/zh-cn/web/svg/element/font-face-format/index.md (hunk 1 above)
Comment (jasonren0403) [reply files/zh-cn/web/svg/element/font-face-format/index.md:5]: image
Let's use the newest commit 👀

**Pull request merged**